lib/generators/provider/templates/spec/spec_helper.rb

```diff
@@ -2,12 +2,17 @@
   require 'simplecov'
   SimpleCov.start
 end
-
+<% if options[:vcr] %>
+VCR.configure do |config|
+  config.ignore_hosts 'codeclimate.com' if ENV['CI']
+  config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
+end
+<% else %>
 # Uncomment in case you use vcr cassettes
 # VCR.configure do |config|
 #   config.ignore_hosts 'codeclimate.com' if ENV['CI']
 #   config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
 # end
-
+<% end %>
```

Comment:
I was trying to DRY it up and came with with this awfulness. 😂 😂 😂 

```ruby
<%= unless options[:vcr] %># Uncomment in case you use vcr cassettes<% end %>
<%= unless options[:vcr] %># <% end %>VCR.configure do |config|
<%= unless options[:vcr] %># <% end %>  config.ignore_hosts 'codeclimate.com' if ENV['CI']
<%= unless options[:vcr] %># <% end %>  config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
<%= unless options[:vcr] %># <% end %>end
```